test/internet/test-dgram-multicast-multi-process.js

```diff
@@ -112,7 +112,8 @@ function launchChildProcess() {
                         worker.pid, count);
 
           assert.strictEqual(count, messages.length,
-                             'A worker received an invalid multicast message');
+                             'A worker received an invalid multicast message' +
+                             `Received ${messages.length}, should be ${count}`);
```

Comment:
If I'm not wrong it should be the opposite: `Received ${count}, should be ${messages.length}`.
I also think it makes sense to remove the error message in favor of the default one.

Reply:
I thought count is the expected number and messages.length is the actual number recieved? I'm actually away from my computer until the 23rd (on vacation!) So I can't take a look until then! But I agree taking out the error message would be a good idea!

Reply:
No, it is actually the other way around.

Reply:
Even if reversed the message is still misleading as the assertion is actually run when **all** messages are received by **all** workers, so my suggestion is to get rid of the `message` argument completely and use the default error message.

```js
assert.strictEqual(count, messages.length);
```

Reply:
See https://github.com/nodejs/node/pull/15998#discussion_r145120053. It's the same here.